.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     parameters:
       v:
         type: string
-        default: "8"
+        default: "12"
```

Comment:
This is now checking Node v12 twice. If I remove the duplicate job then CI just hangs.

I tried to add Node 16 for fun but get a message about not having a password ...